packages/reef-knot/package.json

```diff
@@ -50,6 +50,18 @@
     "@reef-knot/types": "2.0.1",
     "@reef-knot/ledger-connector": "4.1.0"
   },
+  "peerDependencies": {
+    "@lidofinance/lido-ui": "^3.18.0",
+    "@tanstack/react-query": "^5.51.1",
+    "@types/ua-parser-js": "0.7.39",
+    "react": ">=18",
+    "react-dom": ">=18",
+    "react-is": ">=18",
+    "styled-components": "^5.3.5",
+    "ua-parser-js": "1.0.37",
```

Comment:
Why is it a peerdep?

Reply:
https://github.com/search?q=repo%3Alidofinance%2Freef-knot%20ua-parser-js&type=code
seems this can a be local dep for a wallets package